src/napari_brainbow_diagnose/napari.yaml

```diff
@@ -26,3 +29,6 @@ contributions:
       display_name: Diagnose Brainbow Image
     - command: napari-brainbow-diagnose.tooltip_pointer_widget
       display_name: Brainbow Tooltip
+    - command: napari-brainbow-diagnose.read_csv_widget
```

Comment:
I think you can reorder the widget list so your contribution comes first (It should be first step as it is a loading step)